pkg/operatorsource/poller.go

```diff
@@ -67,7 +67,10 @@ type poller struct {
 
 func (p *poller) Initialize() {
 	log.Info("[sync] sending initial package update notification on start.")
-	p.refresher.SendRefresh()
+	for _, opSrcKey := range p.datastore.GetAllOperatorSources() {
+		log.Debugf("[sync] Refreshing OperatorSource %s", opSrcKey.Name.Name)
+		p.refresher.SendRefresh(opSrcKey.Name.Name)
```

Comment:
So the sync behavior is changing where you are now sending a refresh per `OperatorSource`. How did you test this?

Reply:
There were a few checks that needed to be performed.

I added some logging to make sure that if the `csc.spec.source != OpSrcName` that the CSC wasn't updated. Next, I made sure that packages would be removed by creating a CSC that has a source specified but a package name that doesn't exist:
```
apiVersion: "operators.coreos.com/v1"
kind: "CatalogSourceConfig"
metadata:
  name: "source-missing-packages"
  namespace: "openshift-marketplace"
spec:
  source: community-operators
  targetNamespace: openshift-operators
  packages: jaeger,camel-k,missing-package
```

This `CSC` was originally placed in the `Configuring` phase and alerted the user that there was a `missing-package`. After the sync kicked off, the `missing-package` was removed.

Reply:
Please test the regular happy path